github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,7 +15,9 @@ Linked PRs: (links to corresponding PRs, optional)
 ## TODOs:
 - [ ] +1
 - [ ] Passes tests
+- [ ] "How to test" steps have been run through
```

Comment:
I actually feel like this is a checklist for the author, not the reviewer.... so does this make sense here?  The assumption (as Karl points out) is that the reviewer uses the How to Test section and tests.